aries_cloudagent/resolver/base.py

```diff
@@ -132,7 +132,12 @@ async def supports(self, profile: Profile, did: str) -> bool:
 
         return bool(supported_did_regex.match(did))
 
-    async def resolve(self, profile: Profile, did: Union[str, DID]) -> dict:
+    async def resolve(
+        self,
+        profile: Profile,
+        did: Union[str, DID],
+        accept: Optional[Sequence[Text]] = None,
```

Comment:
I like the idea of the DID resolver receiving the accept options, I wonder if we can push that idea up to other levels (like the universal resolver).

Reply:
[Here](https://github.com/decentralized-identity/universal-resolver/blob/main/README.md), I found the following:
```
curl -H "Accept: application/did+ld+json" https://dev.uniresolver.io/1.0/identifiers/did:sov:WRfXPg8dantKVubE3HX8pw
curl -H "Accept: application/did+json" https://dev.uniresolver.io/1.0/identifiers/did:sov:WRfXPg8dantKVubE3HX8pw
curl -H "Accept: application/did+cbor" https://dev.uniresolver.io/1.0/identifiers/did:sov:WRfXPg8dantKVubE3HX8pw
```

To use `Accept` in header but I don't think it will process something `didcomm/aip1`, `didcomm/aip2;env=rfc19` which `pydid` [Indy ledger] can.

Reply:
Right, it's using mime types that would describe the whole document, while we are using one that describes the required service endpoint(s). Maybe it should be called `service_accept`.